components/Navigation/style.css

```diff
@@ -0,0 +1,34 @@
+.header_menu{
+  display: flex;
+  align-items: center;
+  flex-direction: row !important;
+}
+
+.header_component {
+  background: var(--primary-color);
+  width: 100%;
+  height: 3.125rem;
+  align-items: center;
+  display: flex;
+  padding: 0px 0.625rem;
+  justify-content: space-between;
+  color: #ffff;
+  display: flex;
```

Comment:
Here you have used display 2 times, you can avoid these kinds of mistakes by writing properties related to each other in groups like:

```
width: 100%;
height: 3.125rem;

display: flex;
flex-direction: row;
justify-content: space-between;
```